rpms/SPECS/3.12.0/wazuh-manager-3.12.0.spec

```diff
@@ -0,0 +1,1022 @@
+%dir %attr(750, ossec, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/12/04
+%attr(640, root, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/12/04/*
+%dir %attr(750, ossec, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/14
+%dir %attr(750, ossec, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/14/04
+%attr(640, root, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/14/04/*
+%dir %attr(750, ossec, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/16
+%dir %attr(750, ossec, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/ubuntu/16/04
+%dir %attr(750, ossec, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/windows
+%attr(640, root, ossec) %config(missingok) %{_localstatedir}/ossec/tmp/sca-%{version}-%{release}-tmp/windows/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/var
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/db
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/db/agents
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/download
+%dir %attr(770, ossec, ossec) %{_localstatedir}/ossec/var/multigroups
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/run
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/selinux
+%attr(640, root, ossec) %{_localstatedir}/ossec/var/selinux/*
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/upgrade
+%dir %attr(770, root, ossec) %{_localstatedir}/ossec/var/wodles
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/aws
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/aws/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/azure
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/azure/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/docker
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/docker/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap.*
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/template*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content
+%attr(640, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content/*
+
+%{_initrddir}/*
+
+%changelog
+* Mon Oct 18 2019 support <info@wazuh.com> - 3.12.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Mon Oct 7 2019 support <info@wazuh.com> - 3.11.0
+- More info: https://documentation.wazuh.com/current/release-notes/
+* Tue Sep 24 2019 support <info@wazuh.com> - 3.10.3
+- More info: https://documentation.wazuh.com/current/release-notes/
```

Comment:
Remove `3.10.3` version from changelog please.

Reply:
Done in 7f378b0